Gemfile

```diff
@@ -12,4 +12,5 @@ gem "manageiq-style"
 gem "rake", "~> 13.0"
 gem "rspec"
 gem "rubocop"
+gem 'simplecov', require: false
```

Comment:
this should be a dev dependency in the gemspec (in fact all of these should - not sure why they aren't)

Reply:
Also simplecov needs a minimum for codeclimate - looking at another gem, it's

```
spec.add_development_dependency "simplecov", ">= 0.21.2"
```